src/file-utils.ts

```diff
@@ -3,27 +3,57 @@ import path from 'node:path';
 import { promises as stream } from 'node:stream';
 
 import { spawn, ExitCodeError } from '@malept/cross-spawn-promise';
+import { minimatch } from 'minimatch';
 
 const MACHO_PREFIX = 'Mach-O ';
 
+const UNPACKED_ASAR_PATH = path.join('Contents', 'Resources', 'app.asar.unpacked');
+
 export enum AppFileType {
   MACHO,
   PLAIN,
   INFO_PLIST,
   SNAPSHOT,
   APP_CODE,
+  SINGLE_ARCH,
 }
 
 export type AppFile = {
   relativePath: string;
   type: AppFileType;
 };
 
+export type GetAllAppFilesOpts = {
+  singleArchFiles?: string;
+};
```

Comment:
nit: just to make it clear where this property is coming from:

```suggestion
export type GetAllAppFilesOpts = Pick<MakeUniversalOpts, 'singleArchFiles'>;
```

Reply:
Not on the scale of this small module, but I try to avoid them in big codebases since TS build times grow unpredictably when you introduce large dependency cycles through this.